docs/e2e/rm-multi-bench-h3.md

```
@@ -0,0 +1,143 @@
+  bash -c 'echo '"'"'{"db_host":"mariadb","redis_cache":"redis://redis-cache:6379","redis_queue":"redis://redis-queue:6379","redis_socketio":"redis://redis-queue:6379"}'"'"' > sites/common_site_config.json'
+docker exec -w /workspace/e2e-mb-bench2 "$CONTAINER" \
+  bench new-site e2e-mb-site2.localhost --mariadb-root-password 123 --db-root-password 123 --admin-password admin
+
+# 3. Add /workspace to custom search paths so the second bench is discovered
+HOME=$E2E_HOME uv run cwcli config add-path /workspace
+HOME=$E2E_HOME uv run cwcli inspect cwcle2e-mb --update
+```
+
+## Verify both benches are visible
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)